tensornetwork/tn_keras/dense.py

```diff
@@ -19,10 +19,12 @@ class DenseDecomp(Layer):
   # as first layer in a sequential model:
   model = Sequential()
   model.add(
-    DenseDecomp(512, decomp_size=128, activation='relu', input_dim=1024))
+    DenseDecomp(512, decomp_size=128, activation='relu', input_shape=(1024,)))
   # now the model will take as input arrays of shape (*, 1024)
```

Comment:
Minor notes on this comment:

-> input_shape() effectively declares an input to the model, not the layer - it's maybe easier to think about things in that way. You can alternatively avoid the whole question by using Input((1024,) as the first layer in your example :)
-> I wouldn't mention input_dim at all - we shouldn't be encouraging folks to use deprecated APIs.

I think it might be better to put the input_dim note as a comment in the __init__ function, so that we're making the API as simple as possible for new users.

Reply:
Sounds good - I removed mentions of input_dim except for a comment in init.